src/components/calendars/Calendars.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import firebase from "../../config";
+import DispatchGroup from '../components/DispatchGroup';
+
+class Calendars extends Component {
```

Comment:
make sure to implement every component in functional component instead of class component